book/src/dev/zebra-dependencies-for-audit.md

```diff
@@ -0,0 +1,146 @@
+# Zebra dependencies
+
+<span style="color:red">red</span>: Crates that we want to be audited fully.
+<span style="color:yellow">red</span>: Crates we want to be audited with conditions.
+
+## Crates
+
+| Name | Version | Notes
+|------| ------- | -----
+| [<span style="color:red">tower-batch</span>](https://github.com/ZcashFoundation/zebra/tree/main/tower-batch) | [0.2.31](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/tower-batch) |
+| [<span style="color:red">tower-fallback</span>](https://github.com/ZcashFoundation/zebra/tree/main/tower-fallback) | [0.2.31](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/tower-fallback) | 
+| [<span style="color:red">zebra-chain</span>](https://github.com/ZcashFoundation/zebra/tree/main/zebra-chain) | [1.0.0-rc.0](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-chain) |
+| [<span style="color:red">zebra-consensus</span>](https://github.com/ZcashFoundation/zebra/tree/main/zebra-consensus) | [1.0.0-rc.0](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-consensus) |
+| [<span style="color:red">zebra-network</span>](https://github.com/ZcashFoundation/zebra/tree/main/zebra-network) | [1.0.0-rc.0](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-network) |
+| [<span style="color:red">zebra-node-services</span>](https://github.com/ZcashFoundation/zebra/tree/main/zebra-node-services) | [1.0.0-rc.0](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-node-services)  |
+| [<span style="color:red">zebra-rpc</span>](https://github.com/ZcashFoundation/zebra/tree/main/zebra-rpc) | [1.0.0-rc.0](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-rpc) |
+| [<span style="color:red">zebra-script</span>](https://github.com/ZcashFoundation/zebra/tree/main/zebra-script) | [1.0.0-rc.0](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-script) |
+| [<span style="color:red">zebra-state</span>](https://github.com/ZcashFoundation/zebra/tree/main/zebra-state) | [1.0.0-rc.0](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-state) |
+| [<span style="color:red">zebrad</span>](https://github.com/ZcashFoundation/zebra/tree/main/zebrad) | [1.0.0-rc.0](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebrad) |
+
```

Comment:
The `zebra-checkpoints` utility implements the "settled network upgrade" consensus rule, so it needs to be audited:
https://github.com/ZcashFoundation/zebra/tree/main/zebra-utils/src/bin/zebra-checkpoints

> A network upgrade is settled on a given network when there is a social consensus that it has activated with a given activation block hash. A full validator that potentially risks Mainnet funds or displays Mainnet transaction information to a user MUST do so only for a block chain that includes the activation block of the most recent settled network upgrade, with the corresponding activation block hash. Currently, there is social consensus that Canopy has activated on the Zcash Mainnet and Testnet with the activation block hashes given in § 3.12 ‘Mainnet and Testnet’ on p. 20.

https://zips.z.cash/protocol/protocol.pdf#blockchain

Reply:
Yea, i agree, i added this at https://github.com/ZcashFoundation/zebra/pull/5395/commits/603977803afba11ab8d70103dceafb0871768f59